lisa/sut_orchestrator/azure/features.py

```diff
@@ -689,21 +689,48 @@ def setup_rdma(self) -> None:
         if len(devices) > 1:
             # upgrade waagent to latest version to resolve
             # multiple ib devices not getting ip address issue
-            waagent.upgrade_from_source()
+
+            # Upgrade to v2.9.0.4 since the latest v2.9.1.1 version
+            # does not successfully assign IP over the IB interface
+            waagent.upgrade_from_source("v2.9.0.4")
```

Comment:
Forget to mention, please mark the node to dirty, so the environment won't be used by other test cases.

Reply:
Fixed